R/getQuote.R

```diff
@@ -373,19 +369,9 @@ getQuote.av <- function(Symbols, api.key, ...) {
     }
     r <- rbind(r, batch.result)
   }
-
-  colnames(r) <- gsub("(^[[:alpha:]])", "\\U\\1", colnames(r), perl = TRUE)
-  r[, "Trade Time"] <- r[, "LastSaleTimestamp"]
-  r[, "LastSaleTimestamp"] <- NULL
-
-  # merge join to produce empty rows for missing results, so that
-  # return value has the same number of rows and order as the input
-  if(length(Symbols) != NROW(r)) {
-    allSymbols <- data.frame(Ticker = Symbols, stringsAsFactors = FALSE)
-    r <- merge(allSymbols, r, by = "Ticker", all.x = TRUE)
-  }
-
-  rownames(r) <- r$Ticker
-  std.cols <- c("Trade Time", "Open", "High", "Low", "Last", "Volume")
-  return(r[, c(std.cols, setdiff(colnames(r), c(std.cols, "Ticker")))])
+  
+  #Normalize column names and output
+  r <- r[, c("ticker", "lastsaleTimeStamp", "open", "high", "low", "last", "volume")]
```

Comment:
"lastsaleTimeStamp" is a typo. It should be "lastSaleTimeStamp".

Reply:
i just tested this again with 28b581b and i now get ` Error in `[.data.frame`(r, , c("ticker", "lastSaleTimestamp", "open",  :   undefined columns selected `

not sure why we're getting different results on this, but i'm definitely seeing a lower case s

Reply:
i'm actually seeing `lastsaleTimeStamp`, which is different from both commits. I'll push a commit for just that.